flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/xa/JdbcXaSinkFunctionState.java

```diff
@@ -28,6 +30,7 @@
 
 /** Thread-safe (assuming immutable {@link Xid} implementation). */
 @ThreadSafe
+@PublicEvolving
```

Comment:
The `JdbcXaSinkFunctionState` should not mark as `PublicEvolving` due to `XaSinkStateHandler` is `@Internal` now. We should change it to `@Internal`

Reply:
done

Reply:
It seems it hasn't been updated yet?

Reply:
sorry, updated..